install.py

```diff
@@ -594,7 +674,7 @@ def driver():
         dest="arch",
         action="store",
         required=False,
-        default="NATIVE",
+        default="all-major",
```

Comment:
It looks like this argument no longer accepts codenames, e.g. "volta"?

Reply:
Yeah, CMake (and nvcc) accept `native` and not `NATIVE`. The uppercase version was a rapids-cmake thing, but now we have support directly in CMake and nvcc.

Reply:
OK, I'll switch to using "native". But what about the codenames ("volta", "turing" etc.)? Should we switch to using architecture numbers instead?

Reply:
@manopapad once we're using the devcontainers, I'd actually recommend using `all-major`. The object files are sensitive to the CUDA archs list, so if we're building with the same archs list, we'll all be populating the cache for each other/CI.

The devcontainers also come with some custom tuning to automatically parallelize device compilations (at the expense of total parallelism), so recompiling an individual object for all the archs completes in roughly the same time as building it for one arch.

Reply:
@manopapad are you sure the `--arch` flag doesn't work? This just compiled successfully for me:

```shell
~/legate/install.py --cuda --arch ampere --openmp --verbose --editable --clean
```

Reply:
I pulled the most recent version of this branch and `--arch` doesn't work for me, the same error manolis is seeing

Reply:
You are right, it works now.